### PR TITLE
Add note for longhorn-system upgrade for v2 volumes

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn/longhorn#7025

### CODE REVIEW

**Comment by derekbit** [comment]
cc @jillian-maroket @shuo-wu

**Review by innobead — APPROVED** [review]
LGTM

File: content/docs/1.5.0/deploy/important-notes/index.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
 
 Please ensure your Kubernetes cluster is at least v1.21 before upgrading to Longhorn v{{< current-version >}} because this is the minimum version Longhorn v{{< current-version >}} supports.
 
+### Detach All V2 Volumes Before Upgrade
+
+Please note that Longhorn does not support the upgrade when v2 volumes are attached. Prior to initiating the upgrade process, ensure that all v2 volumes are detached.
```
Comment (jillian-maroket) [inline content/docs/1.5.0/deploy/important-notes/index.md:17]:
```suggestion
### Upgrade Prerequisite for Systems Using V2 Volumes

You cannot upgrade Longhorn when volumes using the V2 Data Engine are still attached. Ensure that all V2 volumes are detached before initiating the upgrade process.
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if the changes work.

**Pull request merged**